flang/lib/Lower/Coarray.cpp

```diff
@@ -27,27 +27,27 @@ void Fortran::lower::genChangeTeamConstruct(
     Fortran::lower::AbstractConverter &converter,
     Fortran::lower::pft::Evaluation &,
     const Fortran::parser::ChangeTeamConstruct &) {
-  TODO(converter.genLocation(), "CHANGE TEAM construct");
```

Comment:
Should `genLocation` without an argument be removed?

Reply:
It is still used in one place as a fallback. I renamed it to `genUnknownLocation()` so that it cannot be used by mistake.